staging/src/kubevirt.io/api/usb/v1alpha1/types.go

```diff
@@ -0,0 +1,50 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NodeConfig represents a subset of Node resources that we want to expose
+// to virtual machines.
+//
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +genclient
+// +genclient:nonNamespaced
+type NodeConfig struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              NodeConfigSpec   `json:"spec" valid:"required"`
+	Status            NodeConfigStatus `json:"status,omitempty"`
+}
+
+type NodeConfigSpec struct {
+	USB USB `json:"usb,omitempty"`
+}
+
+// USB defines a group of USB devices based on its selectors and combine them under a single
+// resource name, to be allocated to the VM
+type USB struct {
+	// Identifies the list of USB host devices.
+	// e.g: kubevirt.io/storage, kubevirt.io/bootable-usb, etc
+	ResourceName string `json:"resourceName"`
+	// +listType=atomic
+	USBHostDevices []USBHostDevices `json:"usbHostDevices,omitempty"`
```

Comment:
Maybe `USBHostDevices` -> `Selectors` would clarify this more.

Reply:
Ok